packages/encrypted-blockstore/package.json

```diff
@@ -3,55 +3,28 @@
   "version": "0.18.0",
   "description": "Content-addressed blockstore with pluggable CRDT storage connectors.",
   "type": "module",
-  "main": "./dist/lib/index.cjs",
-  "module": "./dist/lib/index.js",
-  "browser": "./dist/lib/index.global.js",
-  "types": "./dist/lib/index.d.ts",
+  "main": "./src/index.ts",
   "files": [
     "dist/lib"
   ],
   "exports": {
     ".": {
-      "import": "./dist/lib/index.js",
-      "require": "./dist/lib/index.cjs",
-      "types": "./dist/lib/index.d.ts",
-      "script": "./dist/lib/index.global.js",
-      "default": "./dist/lib/index.js"
+      "import": "./src/index.ts"
     },
     "./crypto-node": {
-      "import": "./dist/lib/crypto-node.js",
-      "require": "./dist/lib/crypto-node.cjs",
-      "types": "./dist/lib/crypto-node.d.ts",
-      "script": "./dist/lib/crypto-node.global.js",
-      "default": "./dist/lib/crypto-node.js"
+      "import": "./src/crypto-node.ts"
     },
     "./crypto-web": {
-      "import": "./dist/lib/crypto-web.js",
-      "require": "./dist/lib/crypto-web.cjs",
-      "types": "./dist/lib/crypto-web.d.ts",
-      "script": "./dist/lib/crypto-web.global.js",
-      "default": "./dist/lib/crypto-web.js"
+      "import": "./src/crypto-web.ts"
     },
     "./store-memory": {
-      "import": "./dist/lib/store-memory.js",
-      "require": "./dist/lib/store-memory.cjs",
-      "types": "./dist/lib/store-memory.d.ts",
-      "script": "./dist/lib/store-memory.global.js",
-      "default": "./dist/lib/store-memory.js"
+      "import": "./src/store-memory.ts"
     },
     "./store-node": {
-      "import": "./dist/lib/store-node.js",
-      "require": "./dist/lib/store-node.cjs",
-      "types": "./dist/lib/store-node.d.ts",
-      "script": "./dist/lib/store-node.global.js",
-      "default": "./dist/lib/store-node.js"
+      "import": "./src/store-node.ts"
     },
     "./store-web": {
-      "import": "./dist/lib/store-web.js",
-      "require": "./dist/lib/store-web.cjs",
-      "types": "./dist/lib/store-web.d.ts",
-      "script": "./dist/lib/store-web.global.js",
-      "default": "./dist/lib/store-web.js"
+      "import": "./src/store-web.ts"
```

Comment:
🫡

Reply:
haha, this might be short lived! just makes debugging easier